doc/user/INSTALLATION.md

```diff
@@ -1,20 +1,43 @@
 # Installation
 
-You can find the binary release packages for Windows, Linux and macOS on the [releases page](https://github.com/f3d-app/f3d/releases).
+## Current release packages (1.3.1)
+
+| Platform | Files |
+| -------- | ----- |
+| Windows Installer | [f3d-1.3.1-win64.exe](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-win64.exe) |
+| Windows (Portable) | [f3d-1.3.1-win64.zip](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-win64.zip) |
+| MacOS Installer | [f3d-1.3.1-Darwin.dmg](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-Darwin.dmg) |
+| Debian-based Linux Installer | [f3d-1.3.1-Linux.deb](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-Linux.deb) |
```

Comment:
Debian package ?

Reply:
It works on Ubuntu too

Reply:
Debian-based package ?

Reply:
What i mean is that it is not an installer.